sup3r/pipeline/slicer.py

```diff
@@ -548,19 +566,22 @@ def check_boundary_slice(self, unpadded_slices, cropped_slices, dim):
         lr_slice_stop = unpadded_slices[-1].stop or self.coarse_shape[dim]
 
         # last slice adjustment
-        if 2 * self.spatial_pad + (lr_slice_stop - lr_slice_start) < 4:
+        if (
+            2 * padding + (lr_slice_stop - lr_slice_start)
+            <= self.min_width[dim]
```

Comment:
Personally, i find it really helpful to define some of these weird conditionals, both for readability and also to reduce this crazy one-liner. What are these? Just guessing but i'd do:

```
slice_width = lr_slice_stop - lr_slice_start
padded_width = 2 * padding + slice_width
too_small = padded_width <= self.min_width[dim]
if too_small:
    ...
```
And also wait i would think that if we're `padded_width == min_width` we should be okay based on the docstring for "min_width"? Surprised that it's <=